app/components/icons/markerIcon.tsx

```diff
@@ -1,20 +1,36 @@
 import React from "react";
 
+import { CategoryEnum } from "@/utils/types";
+
 import * as Icons from "./icons";
 
-const categoryIcons: Record<string, React.ComponentType<{ className?: string }>> = {
-  auditorium: Icons.Auditorium,
-  bath: Icons.Wc,
-  water: Icons.Water,
-  food_lunch: Icons.Restaurant,
-  library: Icons.Library,
-  studyroom: Icons.Studyroom,
-  sports_place: Icons.Sport,
-  parking: Icons.Parking,
-  computers: Icons.Print,
+const categoryIcons: Record<CategoryEnum, React.ComponentType<{ className?: string }>> = {
```

Comment:
```suggestion
const categoryIcons: Record<Category, React.ComponentType<{ className?: string }>> = {
```
Si cambia lo de los enums